bundles/org.openhab.binding.webthing/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="webthing"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="generic">
+		<label>WebThing Binding Thing</label>
+		<description>Sample thing for WebThing Binding</description>
+
+		<config-description>
+			<parameter name="webThingURI" type="text" required="true">
```

Comment:
You could specify `<context>url</context>` to get a free validation.

Reply:
thanks for the remark. I have fixed this